pkg/live/inventoryrg.go

```diff
@@ -44,7 +44,7 @@ const (
 // resource used to store inventory.
 var ResourceGroupGVK = schema.GroupVersionKind{
 	Group:   "kpt.dev",
-	Version: "v1alpha1",
+	Version: "v1beta1",
```

Comment:
What is the behavior expected during this version transition ?

For ex. How will previous and new version of kpt deals with packages that contains `alpha` version of resourcegroup in filesystem as well in cluster.

Reply:
The schema for v1alpha1 and v1beta1 are the same. I don't expect users put ResourceGroup CR in their package. The inventory id/label is in the local package, that should be enough for the `kpt live apply`. For clusters with previously installed CRD, I think we need to update the CRD to this new one.

Reply:
This is a good question. We may need to automatically re-apply the CRD during future applies.

Reply:
Just to confirm a user-story:
As a user, I have applied kpt package using `kpt live apply`, so that the ResourceGroup CR in the cluster will be using `v1alpha` version (API as well as storage).
With this change, when I do `kpt live apply`, will the user encounter a failure ?

How will they know what are they suppose to do next ?
Also, to update the CRD definition, they may need higher permissions.

We need to think more about making this transition smoother. Ideal outcome will be that their existing `kpt live apply` continue to work with the `v1alpha` version.